test/e2e/specs/footer/chaise-config.js

```diff
@@ -0,0 +1,13 @@
+var chaiseConfig = {
+    name: "Footer",
+    defaultTables: {
+        "1": {
+            "schema": "isa",
+            "table": "dataset"            
```

Comment:
This is a catalog that only exists on `dev.isrd`, we don't test against this catalog in travis. You shouldn't need to define this property for the footer tests.